packages/shared/src/lib/localLanguage.ts

```diff
@@ -1,50 +1,41 @@
 export const LocalLanguageBannerTemplateName = 'EuropeMomentLocalLanguageBanner';
-const LocalLanguageBannerTestName = 'LOCAL-LANGUAGE';
-const LocalLanguageBannerVariant = 'CONTROL';
-const LocalLanguageEpicTestName = 'LOCAL-LANGUAGE';
-const LocalLanguageEpicVariant = 'CONTROL';
 
 export type LocalLanguage = {
+    testName: string;
+    variantName: string;
     bannerHeader?: string;
-    epicHeader?: string;
-    epicParagraphs?: string[];
-    epicHighlightedText?: string;
 };
 type LocalLanguages = Record<string, LocalLanguage>;
 
 const localLanguages: LocalLanguages = {
     FR: {
+        testName: 'LOCAL-LANGUAGE',
+        variantName: 'CONTROL',
         bannerHeader: 'Soutenez un journalisme européen et indépendant ',
```

Comment:
Once the `testName` and `variantName` keys are removed this could look something like:

```
const localLanguageBannerHeaders: Record<string, BannerHeaderCopy> = {
	FR: {
		bannerHeader: 'Soutenez un journalisme européen et indépendant ',
	},
	DE: {
		bannerHeader: 'Unterstützen Sie unabhängigen europäischen Journalismus',
	},
	...
```

Reply:
Ok, changed